src/api/public/apidocs-new/paths/build_project_repository_architecture_package_file.yaml

```diff
@@ -0,0 +1,142 @@
+get:
+  summary: This endpoint returns a specific artifact file contents
+  description: |
+    Given a project, repository, architecture and package, retrieve the given file's content.
+  operationId: getBuildProjectRepositoryArchitecturePackageFile
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/repository_name.yaml'
+    - $ref: '../components/parameters/architecture_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - $ref: '../components/parameters/file_name.yaml'
+    - name: view
```

Comment:
@danidoni hmm, if you provide this parameter, the endpoint returns a different schema, also the description doesn't quite match anymore. I think this needs to be defined as separate endpoint

Reply:
I've extracted it into a new endpoint